Server/VikingVault.API/Controllers/PaymentTogglingController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VikingVault.Services;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.API.Controllers
+{
+   
+    [Route("[controller]")]
+    [ApiController]
+    public class PaymentTogglingController : ControllerBase
+    {
+        private readonly IPaymentTogglingService _paymentTogglingService;
+
+        public PaymentTogglingController(IPaymentTogglingService paymentTogglingService)
+        {
+            _paymentTogglingService = paymentTogglingService;
+        }
+
+        [HttpGet("{id}", Name = "Get")]
+        public ActionResult<Boolean> Get(int id)
+        {
+            Console.Write(id);
+            bool? isPaymentEnabled = _paymentTogglingService.IsPaymentEnabled(id);
+            if (isPaymentEnabled == null)
+                return StatusCode(500, "Internal Server Error");
+            if (isPaymentEnabled == true)
```

Comment:
`return Ok(isPaymentEnabled)` is the short way to get rid of this if/else, and it achieves exactly the same functionality